src/components/DropdownList.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import '../App.css';
+import StationModel from '../models/StationModels';
+import StationListItem from './StationListItem';
+
+interface DropdownProps<T> {
```

Comment:
Really like the idea of keeping the dropdown generic with generic types so it can be resused, here it wouldn't be resuable as is, as the `DropdownList` component specified the `StationModel` and `StationListItem`, so it would need to be refactored a bit to be reused.

No worries about this now though unless you end up needing it to be reused.